Samples/MatrixMultiply/Program.cs

```diff
@@ -140,25 +145,23 @@ static void RunMatrixMultiply(float[,] a, float[,] b, float[,] expectedResult)
             Console.WriteLine($"- Naive implementation: {sw.ElapsedMilliseconds}ms");
 
             // Accelerated implementations
-            using (var context = Context.CreateDefault())
+            using var context = Context.CreateDefault();
+
+            foreach (var device in context)
             {
-                foreach (var device in context)
-                {
-                    using (var accelerator = device.CreateAccelerator(context))
-                    {
-                        sw.Restart();
-                        var acceleratedResult = MatrixMultiplyAccelerated(accelerator, a, b);
-                        sw.Stop();
-                        Debug.Assert(MatrixEqual(acceleratedResult, expectedResult));
-                        Console.WriteLine($"- Accelerated implementation on {accelerator}: {sw.ElapsedMilliseconds}ms");
-
-                        sw.Restart();
-                        var acceleratedTiledResult = MatrixMultiplyTiled(accelerator, a, b);
-                        sw.Stop();
-                        Debug.Assert(MatrixEqual(acceleratedTiledResult, expectedResult));
-                        Console.WriteLine($"- Tiled implementation on {accelerator}: {sw.ElapsedMilliseconds}ms");
-                    }
-                }
+                using var accelerator = device.CreateAccelerator(context);
+
+                sw.Restart();
+                var acceleratedResult = MatrixMultiplyAccelerated(accelerator, a, b);
+                sw.Stop();
```

Comment:
@MoFtZ @mikhail-khalizev Idea: Maybe we should add add an additional sample to demonstrate the newly added profiling capabilities in the future?

Reply:
Added new `ProfilingMarkers` sample project.